packages/analytics-browser/src/browser-client.ts

```diff
@@ -124,200 +125,38 @@ export class AmplitudeBrowser extends AmplitudeCore<BrowserConfig> {
- * console.log(result.message); // "Event tracked successfully"
- * ```
- */
-export const identify = returnWrapper(client.identify.bind(client));
-
 /**
- * Sends a group identify event containing group property operations.
- *
- * ```typescript
- * const id = new Identify();
- * id.set('skills', ['js', 'ts']);
- * const groupType = 'org';
- * const groupName = 'engineering';
- * groupIdentify(groupType, groupName, id);
+ * > EXPERIMENTAL
  *
- * // alternatively, this tracking method is awaitable
- * const result = await groupIdentify(groupType, groupName, id).promise;
- * console.log(result.event); // {...}
- * console.log(result.code); // 200
- * console.log(result.message); // "Event tracked successfully"
- * ```
- */
-export const groupIdentify = returnWrapper(client.groupIdentify.bind(client));
-export const setGroup = returnWrapper(client.setGroup.bind(client));
-
-/**
- * Sends a revenue event containing revenue property operations.
+ * Creates additional Amplitude instance. This can be used to instantiate
+ * a new tracker with a different `API_KEY` and configuration.
  *
  * ```typescript
- * const rev = new Revenue();
- * rev.setRevenue(100);
- * revenue(rev);
- *
- * // alternatively, this tracking method is awaitable
- * const result = await revenue(rev).promise;
- * console.log(result.event); // {...}
- * console.log(result.code); // 200
- * console.log(result.message); // "Event tracked successfully"
- * ```
- */
-export const revenue = returnWrapper(client.revenue.bind(client));
-
-/**
- * Returns current user ID.
- *
- * ```typescript
- * const userId = getUserId();
- * ```
- */
-export const getUserId = client.getUserId.bind(client);
-
-/**
- * Sets a new user ID.
- *
- * ```typescript
- * setUserId('userId');
- * ```
- */
-export const setUserId = client.setUserId.bind(client);
-
-/**
- * Returns current device ID.
- *
- * ```typescript
- * const deviceId = getDeviceId();
- * ```
- */
-export const getDeviceId = client.getDeviceId.bind(client);
-
-/**
- * Sets a new device ID.
- * When setting a custom device ID, make sure the value is sufficiently unique.
- * A uuid is recommended.
- *
- * ```typescript
- * setDeviceId('deviceId');
- * ```
- */
-export const setDeviceId = client.setDeviceId.bind(client);
-
-/**
- * Returns current session ID.
- *
- * ```typescript
- * const sessionId = getSessionId();
- * ```
- */
-export const getSessionId = client.getSessionId.bind(client);
-
-/**
- * Sets a new session ID.
- * When settign a custom session ID, make sure the value is in milliseconds since epoch (Unix Timestamp).
- *
- * ```typescript
- * setSessionId(Date.now());
- * ```
- */
-export const setSessionId = client.setSessionId.bind(client);
-
-/**
- * Sets a new optOut config value. This toggles event tracking on/off.
- *
- *```typescript
- * // Stops tracking
- * setOptOut(true);
- *
- * // Starts/resumes tracking
- * setOptOut(false);
- * ```
- */
-export const setOptOut = client.setOptOut.bind(client);
-
-/**
```

Comment:
Those comments are well explained, feel we should keep those.
Can we move the comments inside the creteInsance function?